mage_ai/data_cleaner/column_types/column_type_detector.py

```diff
@@ -60,12 +70,21 @@ def find_syntax_errors(series, column_type):
         and not dtype is pd.Timestamp
     ):
         str_series = str_series.astype(str)
+        filter_invalid = True
```

Comment:
actually, for invalid datetime, let's fill in with NaT

Reply:
Invalid datetimes now resolve to NaT